app/components/primer/component.rb

```diff
@@ -141,5 +141,9 @@ def should_raise_error?
     def should_raise_aria_error?
       !Rails.env.production? && raise_on_invalid_aria? && !ENV["PRIMER_WARNINGS_DISABLED"]
     end
+
+    def deprecation
+      ::Primer::ViewComponents.deprecation
+    end
```

Comment:
Why not call `::Primer::ViewComponents.` directly above?

Reply:
Unless you plan to do any extra work in the deprecation call, it seems like an unnecessary abstraction